### PR TITLE
chore: add e2e playwright setup

### PR DESCRIPTION
**Pull request by chinook25**
@MaxPostema and @Fernananas Would like to have a set of minimal expected emx2 functionality smoke tested: this PR seems like a good place to start. 

Our expected minimal set of basic functionality:
https://docs.google.com/spreadsheets/d/19DEH8fcV7tYCkYW0LDeG2kvgpz2bzdVGhpjDGrUehTE/edit#gid=0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "molgenis-emx2",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:molgenis/molgenis-emx2.git",
+  "author": "Joris de Keijser <chinook25@gmail.com>",
```
Comment (connoratrug) [inline package.json:6]: do you want your own adres here ?

File: playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ..devices['Desktop Chrome'], channel: 'chrome' },
+    // },
+  ],
+
+  /* Run your local dev server before starting the tests */
+  // webServer: {
+  //   command: 'npm run start',
```
Comment (mswertz) [inline playwright.config.ts:73]: so if we here would state 'gradle run' then in theory that is all CI would need to run this too?

**Review by mswertz — COMMENTED** [review]
nice, I would like to try to get this going on CI next sprint. Seems to be a small step away see comment on the config file.

File: playwright.config.ts (hunk 2 above)
Comment (chinook25) [reply playwright.config.ts:73]: I think so yea. Just have a running backend and the tests can run too.

**Pull request closed without merging**